lib/pf/radius.pm

```diff
@@ -228,17 +234,16 @@ sub authorize {
         $args->{'isPhone'} = $FALSE;
     }
 
-    #define the current connection value to instantiate the correct portal
-    my $options = {};
-
-    $options->{'last_connection_sub_type'} = $args->{'connection_sub_type'} if (defined( $args->{'connection_sub_type'}));
-    $options->{'last_connection_type'} = connection_type_to_str($args->{'connection_type'}) if (defined( $args->{'connection_type'}));
-    $options->{'last_switch'}          = $switch_id;
-    $options->{'last_port'}            = $args->{'switch'}->{switch_port} if (defined($args->{'switch'}->{switch_port}));
-    $options->{'last_vlan'}            = $args->{'vlan'} if (defined($args->{'vlan'}));
-    $options->{'last_ssid'}            = $args->{'ssid'} if (defined($args->{'ssid'}));
-    $options->{'last_dot1x_username'}  = $args->{'user_name'} if (defined($args->{'user_name'}));
-    $options->{'realm'}                = $args->{'realm'} if (defined($args->{'realm'}));
+    $options->{'connection_sub_type'} = $args->{'connection_sub_type'} if (defined( $args->{'connection_sub_type'}));
+    $options->{'connection_type'}     = connection_type_to_str($args->{'connection_type'}) if (defined( $args->{'connection_type'}));
+    $options->{'switch'}              = $switch_id;
+    $options->{'port'}                = $args->{'switch'}->{switch_port} if (defined($args->{'switch'}->{switch_port}));
+    $options->{'vlan'}                = $args->{'vlan'} if (defined($args->{'vlan'}));
+    $options->{'ssid'}                = $args->{'ssid'} if (defined($args->{'ssid'}));
+    $options->{'dot1x_username'}      = $args->{'user_name'} if (defined($args->{'user_name'}));
```

Comment:
Put back to the 'last_' prefix as I rewrite the field names in ad2dd1eada2c3897103cc99c08e0987fabc6353d to match the profile conditions.